proteus/mprans/SW2DCV.h

```diff
@@ -1604,9 +1512,9 @@ class SW2DCV : public SW2DCV_base {
                          ith_dHij_minus_muHij_times_hvStarStates -
                          ith_muHij_times_hvStates);
           // clean up potential negative water height due to machine precision
-          if (globalResidual[offset_h + stride_h * i] >= -hEps &&
-              globalResidual[offset_h + stride_h * i] < hEps)
-            globalResidual[offset_h + stride_h * i] = 0;
+          // if (globalResidual[offset_h + stride_h * i] >= -hEps &&
```

Comment:
should this just be removed?

Reply:
Yes, I need to clean out some of the commented code.